### ISSUE TITLE
<hgroup> content model should allow <p> elements per HTML Living Standard

### ISSUE THREAD

**Issue by theodorebje** [validator/validator#2061]
### Summary

The checker reports an error when a `<p>` element is used inside an `<hgroup>` element, even though the HTML Living Standard explicitly allows zero or more `<p>` elements before and after a single heading inside `<hgroup>`.

### URL for document that reproduces the problem

`n/a`

### HTML/CSS/SVG snippet that reproduces the problem

```html
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>hgroup False Positive</title>
</head>

<body>
    <hgroup>
        <h1>The reality dysfunction</h1>
        <p>Space is not the only void</p>
    </hgroup>
</body>

</html>
```

### Checker error messages

```shell
Error: Element p not allowed as child of element hgroup in this context. (Suppressing further errors from this subtree.)

From line 12, column 9; to line 12, column 11

>↩        <p>Space 

Contexts in which element p may be used:
    Where flow content is expected.
Content model for element hgroup:
    One or more h1, h2, h3, h4, h5, h6 elements, optionally intermixed with script-supporting elements.
```

### Spec links/citations

https://html.spec.whatwg.org/multipage/sections.html#the-hgroup-element

### Steps to reproduce

`n/a`

### Expected behavior

`n/a`

### Actual behavior

`n/a`

### Screenshots or screen recordings

Tested with both file upload and text input.

<img width="2256" height="2186" alt="Image" src="https://github.com/user-attachments/assets/96ab35e5-8696-446c-af5e-f4186a9c0554" />

**Comment by stevefaulkner** [comment]
yes it should 🙂

**Comment by sideshowbarker** [comment]
See https://validator.w3.org/nu/?showsource=yes&doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFET0NUWVBFIGh0bWw%2BCjxodG1sIGxhbmc9ImVuIj4KCjxoZWFkPgogICAgPG1ldGEgY2hhcnNldD0iVVRGLTgiPgogICAgPHRpdGxlPmhncm91cCBGYWxzZSBQb3NpdGl2ZTwvdGl0bGU%2BCjwvaGVhZD4KCjxib2R5PgogICAgPGhncm91cD4KICAgICAgICA8aDE%2BVGhlIHJlYWxpdHkgZHlzZnVuY3Rpb248L2gxPgogICAgICAgIDxwPlNwYWNlIGlzIG5vdCB0aGUgb25seSB2b2lkPC9wPgogICAgPC9oZ3JvdXA%2BCjwvYm9keT4KCjwvaHRtbD4%3D#textarea

It appears you used https://validator.nu to check your document. Please always use https://validator.w3.org/nu instead. The checker version deployed at https://validator.nu is now more than 4 years out of date. I don’t maintain https://validator.nu and I don’t have any means to update it.
